### PR TITLE
fixes for AAC

### PR DESCRIPTION
**Pull request by OlegHahm**
This PR unset the TENTATIVE flag on the local address entry if appropriate and set the SL2AO in the neighbor solicitation if the source address is specified.

### CODE REVIEW

**Comment by authmillenon** [comment]
Oh, I somehow missed this one...

File: sys/shell/commands/sc_netif.c [hunk 1]
```diff
@@ -713,13 +714,16 @@ static int _netif_add(char *cmd_name, kernel_pid_t dev, int argc, char **argv)
...
+    if ((assigned_address = gnrc_ipv6_netif_add_addr(dev, &addr, prefix_len,
+                                                    (type == _ANYCAST) ?
+                                                    GNRC_IPV6_NETIF_ADDR_FLAGS_NON_UNICAST :
+                                                    GNRC_IPV6_NETIF_ADDR_FLAGS_UNICAST))
+                           == NULL) {
         printf("error: unable to add IPv6 address\n");
         return 1;
     }
+    gnrc_ipv6_netif_addr_t *netif_addr = gnrc_ipv6_netif_addr_get(assigned_address);
+    netif_addr->flags &= ~GNRC_IPV6_NETIF_ADDR_FLAGS_TENTATIVE;
```
Comment (authmillenon) [inline sys/shell/commands/sc_netif.c]: That's not right btw. The tentative state is basically saying "let me first check if someone else has this address". If you don't this for manually configured addresses, you are in hotter waters than when you do it for automatically configured addresses.

Comment (OlegHahm) [inline sys/shell/commands/sc_netif.c]: If you don't do this we will send out an ARO as a router and refuse any response to this address. If I configure the address manually using a debug tool like the shell, I'm supposed to know what I am doing.

Comment (authmillenon) [inline sys/shell/commands/sc_netif.c]: Args I confused IPv6-AAC (probe + timing based) with 6LoWPAN-AAC (probe + answer based). You are totally right.

**Pull request merged**